app/helpers/container_service_helper/textual_summary.rb

```diff
@@ -69,4 +69,48 @@ def textual_port_config(port_conf)
       :value => "#{port_conf.protocol} port #{port_conf.port} to pods on target port:'#{port_conf.target_port}'"
     }
   end
+
+  def textual_ems
+    ems = @record.ext_management_system
+    return nil if ems.nil?
+    label = ui_lookup(:table => "ems_container")
+    h = {:label => label, :image => "vendor-#{ems.image_name}", :value => ems.name}
+    if role_allows(:feature => "ems_container_show")
+      h[:title] = "Show parent #{label} '#{ems.name}'"
+      h[:link]  = url_for(:controller => 'ems_container', :action => 'show', :id => ems)
+    end
+    h
+  end
+
+  def textual_container_groups
+    num_of_container_groups = @record.number_of(:container_groups)
+    label = ui_lookup(:tables => "container_groups")
+    h = {:label => label, :image => "container_group", :value => num_of_container_groups}
+    if  num_of_container_groups > 0 && role_allows(:feature => "container_group_show")
+      h[:link] = url_for(:action => 'show', :controller => 'container_service', :display => 'container_groups')
+    end
+    h
+  end
+
+  def textual_container_project
+    project = @record.container_project
+    return nil if project.nil?
+    label = ui_lookup(:table => "container_project")
+    h = {:label => label, :image => "container_project", :value => project.name}
+    if role_allows(:feature => "container_project_show")
+      h[:title] = "Show parent #{label} '#{project.name}'"
+      h[:link] = url_for(:controller => 'container_project', :action => 'show', :id => project)
+    end
+    h
+  end
+
+  def textual_container_routes
```

Comment:
Remove entirely this method (it has been unified somewhere else so you can just drop it).